scripts/generate_changelog.py

```
@@ -0,0 +1,305 @@
+    """Get the most recent version tag from git."""
+    try:
+        result = subprocess.run(
+            ["git", "describe", "--tags", "--abbrev=0", "--match=v*"],
+            capture_output=True,
+            text=True,
+            check=False,
+        )
+        if result.returncode == 0 and result.stdout.strip():
+            return result.stdout.strip()
+    except Exception:
+        pass
```

Ruff (0.14.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)